challenge.rb

```diff
@@ -45,7 +46,16 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ((ac_working == true) && (current_temp > desired_temp))
+    puts "Turn on the AC."
+  elsif (ac_working == true) && (current_temp <= desire_temp))
+    puts "Just Right!"
+  elsif ((ac_working == false) && (current_temp <= desire_temp))
+    puts "Fix the AC whenever you have the chance. It's cool"
+  else "Fix the AC now! It's hot!"
```

Comment:
Should be returning values, not printing them. Otherwise right (apart from some typos).